tests/issues.targets

```diff
@@ -40,6 +40,9 @@
         <ExcludeList Include="$(XunitTestBinBase)\GC\Scenarios\muldimjagary\muldimjagary\*">
             <Issue>3392</Issue>
         </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)\GC\Scenarios\DoublinkList\dlstack\*">
```

Comment:
It was an artifact of a merge/rebase; hopefully I managed to revert the merge correctly (it's kind of a pain with git). Those files have disappeared from this page so hopefully all is well now ;).